libbeat/kibana/testdata/fields.yml

```diff
@@ -24,6 +24,8 @@
           value: "_a=(query:(query_string:(analyze_wildcard:!t,query:'error.grouping_key:%22{{value}}%22')))"
         - min_version: 6.0.0
           value: "_a=(query:(language:lucene,query:'context.app.name:\"{{value}}\"'))"
+        - min_version: 7.0.0
```

Comment:
The tests don't support this at the moment, best just change the line above with 6.0.0 to kquery instead and tests should pass. We need to cleanup these tests on our end.